types/Yargs.ts

```diff
@@ -40,4 +41,9 @@ export interface YargsCommandModule<T, U> extends CommandModule<T, U> {
   builder: (yargs: Argv) => Promise<Argv<U>>;
 }
 
+export type CommandModuleBucket<T, U> = PartialProp<
+  YargsCommandModule<T, U>,
+  'handler'
+>;
+
```

Comment:
I just realized there is a type right below this for the `YargsCommandModuleBucket`.  Do we want to use that and add handlers?  Or use this and leave them as optional?

Reply:
I kept `handler` on the bucket type because I ran into an issue with nested command buckets. `yargs.command()` gets mad if you pass it something that doesn't include a handler. So for nested buckets, like `hs custom-object schema <something>` the typing for `schema` needs to have a handler. I assume this will also matter once we port cli.js to TS since it calls `yargs.command` on every bucket. Maybe there's another way around this that I'm not seeing though.

Reply:
Gotcha, I'll switch to the other type and add an empty handler